jinja2_fsloader/__init__.py

```diff
@@ -33,10 +33,11 @@ class FSLoader(jinja2.BaseLoader):
 
     """
 
-    def __init__(self, template_fs, encoding='utf-8', use_syspath=False):
+    def __init__(self, template_fs, encoding='utf-8', use_syspath=False, fs_filter=[]):
```

Comment:
Please use `None` as default instead of a literal list. [The rational can be found here](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) Furthermore since `None` is the default for the `filter` argument of a `Walker` there is not even a reason to instantiate a list when given the default argument.

Reply:
Done, I’ve also extended the docstring a bit.